### DISCUSSION TITLE
byedpi больше на работает

### DISCUSSION THREAD

**Discussion by GennadiyVick** [hufrea/byedpi#139] [Q&A]
Привет всем, с 19.09.2024 пошли ошибки, запускаю ./ciadpi -d 1
ошибки :
yt-dlp --proxy socks5://127.0.0.1:1080 -f 140 https://www.youtube.com/watch?v=xxxxxxx
[youtube] Extracting URL: https://www.youtube.com/watch?v=xxxxxxxxxx
[youtube] xxxxxxxxxx: Downloading webpage
[youtube] xxxxxxxxxx: Downloading ios player API JSON
[youtube] xxxxxxxxxx: Downloading web creator player API JSON
[youtube] xxxxxxxxxxI: Downloading m3u8 information
[info] xxxxxxxxxx: Downloading 1 format(s): 140
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (1/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (2/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (3/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (4/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (5/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (6/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (7/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (8/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (9/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (10/10)...
Кто-то знает как решать?

**Comment by dumbodumbos** [comment]
--proto http --auto=none --disorder 1 --auto=torst --tlsrec 1+s & - таким конфигом перестал работать youtube. СПб - Ростелеком.

**Reply by GennadiyVick** [reply]
таким тоже

**Comment by jura1243** [comment]
Подтверждаю. У меня тоже не работает в Санкт-Петербурге в Ростелекоме.

**Comment by vovkkk** [comment]
У меня похожая проблема, тоже в ночь с 19 на 20 посреди просмотра остановилось (примерно в 22-23 по МСК).
Но Ростелеком Новосибирск.

Я использовал команду в `byedpi-12-x86_64-w64`

    ciadpi.exe --disorder 2 --split 1 --tlsrec 1+s

Причём yt по прежнему открывается, коменты, аватарки, превьюхи — всё доступно, всё загружается, кроме воспроизведения видео. Так же по прежнему  открываются рутрекер и ntc.party.

Обновившись на `byedpi-14.1-x86_64-w64`, я пробовал разные опции из комментов: добавлял `--fake`, менял числовые значения — всё без толку.
А сейчас ещё и `--debug 2` перестал работать: в консоли пусто, хотя 5 минут назад оно ещё сыпало кучу сообщений.

Интересно, что в этой же сети Android TV с PowerTunnel показывает YT, и телефон без ничего показывает.
А на ПК даже андроид в виртуалке не показывает, хотя летом когда это начиналось я несколько дней был вынужден смотреть YT через BlueStacks и ByeDPIAndroid.

**Comment by hufrea** [comment]
Что насчет `--oob 1`?
Или `--fake -1 --fake-data tls_clienthello_www_google_com.bin`, ClientHello [отсюда](https://github.com/bol-van/zapret/blob/master/files/fake/tls_clienthello_www_google_com.bin).

**Reply by vovkkk** [reply]
Спасибо! У меня заработало 

    ciadpi.exe --oob 1 --disorder 2 --split 1 --tlsrec 1+s

Единственный минус: приходится качество вручную ставить, по умолчанию YT предлагает 144p

**Reply by dumbodumbos** [reply]
У меня тоже починилось. Спасибо!

**Reply by EgoBrainProgrammer** [reply]
Парни, пожалуйста, свяжитесь со мной через дискорд, нужна консультация, умоляю
predator5386

**Reply by jura1243** [reply]
заработало `/home/jura/byedpi/ciadpi --oob 1 --disorder 2 --split 1 --tlsrec 1+s &`

**Reply by ALV-036** [reply]
На Ростелекоме  заработало в таком формате  ciadpi-x86_64 --disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 1

Хотя только день назад не работало,  Ростелеком  всё играется там с настройками.

**Reply by GennadiyVick** [reply]
ciadpi.exe --oob 1 --disorder 2 --split 1 --tlsrec 1+s
Вчера 21.09 работало , а сегодня с этими параметрами уже не работает

**Reply by dumbodumbos** [reply]
Подтверждаю. Перестало работать. СПб Ростелеком

**Reply by ALV-036** [reply]
Если вы на Windows ставите ПО локально то попробуйте другое ПО, неплохо себя показала   https://5time.taplink.ws/
качайте  5timeDPI (для Windows, новая версия v4.1)   запуск только от имени Администратора.
   Если она перестаёт работать то разработчик уже новую версию выпускает.

  По Byedpi   у меня Linux версия на Ubuntu      и строка ciadpi-x86_64 --disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 1   работает на Ростелеком Воронежская область..    Попробуйте  виртуалку Ubuntu  поднимите, и в ней проверьте, возможно Windows и Linux версии по разному себя ведут.

**Reply by GennadiyVick** [reply]
Неа так тоже не работает
/ciadpi --disorder 1 --fake 0 --ttl 1 -A torst --tlsrec 1+s --debug 1

yt-dlp https://www.youtube.com/watch?v=XXXXXXX --proxy="socks5://127.0.0.1:1080"
[youtube] Extracting URL: https://www.youtube.com/watch?v=XXXXXXX
[youtube] XXXXXXX: Downloading webpage
[youtube] XXXXXXX: Downloading ios player API JSON
[youtube] XXXXXXX: Downloading web creator player API JSON
[youtube] XXXXXXX: Downloading m3u8 information
[info] XXXXXXX: Downloading 1 format(s): 303+251
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (1/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (2/10)...
[download] Got error: _ssl.c:990: The handshake operation timed out. Retrying (3/10)...

**Reply by ALV-036** [reply]
Пробуйте 5timedpi   она отрабатывала даже когда byedpi не работал.

**Reply by hufrea** [reply]
Замена disorder на split не помогает? 
`--oob 1 --split 2`

**Reply by rootk1t** [reply]
> Замена disorder на split не помогает? `--oob 1 --split 2`

работает и ваше решение и --split 1 --oob 1

> Пробуйте 5timedpi она отрабатывала даже когда byedpi не работал.

Зачем вы рекламируйте 5timedpi в треде с ByeDPI?

**Reply by decide-you** [reply]
> Что насчет `--oob 1`? Или `--fake -1 --fake-data tls_clienthello_www_google_com.bin`, ClientHello [отсюда](https://github.com/bol-van/zapret/blob/master/files/fake/tls_clienthello_www_google_com.bin).

@hufrea 
При использовании `ciadpi -i 127.0.0.1 -p 8080 --oob 1` и соответствующей настройке SOCKS5 в Firefox ютуб работает. Росстелеком Новосибирск

**Reply by Nowhaile** [reply]
С `--fake -1 --fake-data tls_clienthello_www_google_com.bin` работает не на всех видео(как я понял, только популярные видео грузит) и порой с задержкой(начинает загружаться через некоторое время после загрузки страницы).
Подобрал `--split 1 --disorder 3+s --fake -1`. Так у меня нормально работает.

**Reply by GennadiyVick** [reply]
пробовал разные варианты предложенные здесь, ни один не работает с 22.09 
OS linux + ростелеком

**Comment by vladiscripts** [comment]
Заработало с настройками [выше](https://github.com/hufrea/byedpi/discussions/139#discussioncomment-10717388).
Но также работает [с настройками с ByeDPIAndroid](https://github.com/dovecoteescapee/ByeDPIAndroid/issues/151): `-s 1 -o 1 -A r -o 1 -A t -f -1 -r 1+s -A s`.

СПб, Ростелеком.

**Comment by Mihrutkin** [comment]
Завелось, пробуйте command: --hosts /host.crack -s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S --debug 1 --auto=none

В host.crack youtube сайты соответственно

**Reply by GennadiyVick** [reply]
./ciadpi --hosts ./host.crack -s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S --debug 1 --auto=none
Если я правильно понял, в файл host.crack нужно поместить ссылку https://www.youtube.com ?

 yt-dlp --proxy="socks5://127.0.0.1:1080" https://www.youtube.com/watch?v=Nn4Ho_kL83U
[youtube] Extracting URL: https://www.youtube.com/watch?v=Nn4Ho_kL83U
[youtube] Nn4Ho_kL83U: Downloading webpage
[youtube] Nn4Ho_kL83U: Downloading ios player API JSON
WARNING: [youtube] [Errno 104] Connection reset by peer. Retrying (1/3)...
[youtube] Nn4Ho_kL83U: Downloading ios player API JSON
[youtube] Nn4Ho_kL83U: Downloading web creator player API JSON
WARNING: [youtube] Remote end closed connection without response. Retrying (1/3)...
[youtube] Nn4Ho_kL83U: Downloading web creator player API JSON
[youtube] Nn4Ho_kL83U: Downloading m3u8 information
WARNING: [youtube] Failed to download m3u8 information: [Errno 104] Connection reset by peer
[info] Nn4Ho_kL83U: Downloading 1 format(s): 308+251
[download] Resuming download at byte 8704

**Reply by Mihrutkin** [reply]
Попробуйте без файла  -s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S --debug 1

**Comment by antonnagorniy** [comment]
Заработал только с таким конфигом, Линукс+Ростелеком:
ciadpi --split 1 --oob 1 --mod-http=h,d --auto none --fake -1 --tlsrec 3+h

**Reply by dumbodumbos** [reply]
> Заработал только с таким конфигом, Линукс+Ростелеком: ciadpi --split 1 --oob 1 mod-http=h,d --auto none --fake -1 --tlsrec 3+h

Спасибо. Так же заработало. Ios устройства + ростелек - работают. А вот сафари на masOC - так же в ошибку впадает. Но достаточно и этого.

**Reply by decide-you** [reply]
@antonnagorniy Разве не `--mod-http=h,d`?

**Reply by bbrvi2008** [reply]
Спасибо, заработало.

**Reply by antonnagorniy** [reply]
> @antonnagorniy Разве не `--mod-http=h,d`?

Да, все верно, куда-то пропали, поправил

**Comment by GennadiyVick** [comment]
УРРААА !!! Завелось вот так:
./ciadpi --oob 1 -s 2 -M h,d -A torst -T 3 -f -1 -r 3+s -S --debug 1 --auto=none
Не знаю, что там за волшебные параметры сработали, но у меня вот так работает.

**Reply by v1rus1** [reply]
Подскажите нубу как эти настройки прописать? windows11. В блокноте сойдет или через cmd как-то надо?

**Reply by chemizt** [reply]
> Подскажите нубу как эти настройки прописать? windows11. В блокноте сойдет или через cmd как-то надо?

@v1rus1 поменяйте аргументы, прописанные в `byedpi.bat`, на указанные @GennadiyVick.
У меня (МГТС, Москва) заработало без `--debug 1` (нужен для отладочного вывода) и `-T 3` (задаёт таймаут ожидания ответа от сервера). В развёрнутом виде запись аргументов у меня выглядит так:
`--oob 1 --split 2 --mod-http h,d --auto torst --fake -1 --tlsrec 3+s --md5sig --auto none`

**UPD:** перепробовал кучу вариантов, но под Windows так и не заработало. Грузится всё, кроме видео.

**Comment by v1rus1** [comment]
на телефоне получилось настроить, а на ПК нет :(

**Comment by Mihrutkin** [comment]
Пробуйте актуальный -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An

**Reply by poiNt3D** [reply]
> Что насчет `--oob 1`? Или `--fake -1 --fake-data tls_clienthello_www_google_com.bin`, ClientHello [отсюда](https://github.com/bol-van/zapret/blob/master/files/fake/tls_clienthello_www_google_com.bin).




> Пробуйте актуальный -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An

Что за прикол? У нас разные версии программы?

```
ciadpi.exe --fake -1 --fake-data tls_clienthello_www_google_com.bin
read/parse: 2
invalid value: -l tls_clienthello_www_google_com.bin

ciadpi.exe -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An
ciadpi.exe: unknown option -- Y
```

**Reply by jura1243** [reply]
> Что за прикол? У нас разные версии программы?
надо указывать в какой системе запускаете.

**Reply by Fidanze** [reply]
Как адаптировать это под андроид? На компе работает без запинки, а на телефоне совсем не фурычит

**Reply by vladiscripts** [reply]
> Как адаптировать это под андроид? На компе работает без запинки, а на телефоне совсем не фурычит

Например [выше](https://github.com/hufrea/byedpi/discussions/139#discussioncomment-10720878) одна настройка и ссылка на другую. У меня работают на компе и смартфоне. не одна так другая.
У [ByeDPIAndroid есть свой форум ](https://github.com/dovecoteescapee/ByeDPIAndroid/discussions)с вариантами настройки.

**Reply by slesru** [reply]
> Пробуйте актуальный -s1 -q1 -Y -Ar -s5 -o1+s -At -f-1 -r1+s -As -s1 -o1 +s -s-1 -An

видео не грузится.

**Reply by Mihrutkin** [reply]
> Как адаптировать это под андроид? На компе работает без запинки, а на телефоне совсем не фурычит

Попробуйте эти настройки и запустить в newpipe

**Reply by Mihrutkin** [reply]
> Как адаптировать это под андроид? На компе работает без запинки, а на телефоне совсем не фурычит

Или ставь версию revanced root и в ней отключи quick

**Reply by EMWD** [reply]
С таким параметрами снова работает, спасибо!
